src/test/java/nl/tudelft/contextproject/controller/ControllerTest.java

```diff
@@ -155,7 +172,11 @@ public Spatial getSpatial() {
 			public void setSpatial(Spatial spatial) { }
 
 			@Override
-			public void mapDraw(Graphics2D g, int resolution) { }			
+			public PhysicsControl getPhysicsObject() {
+				return null;
+			}
+			
+			public void mapDraw(Graphics2D g, int resolution) { }
```

Comment:
Did you just comment 3 times on the same issue? Or is it 3 times more important then the rest of the issues?